cookbook/case_studies/ml_training/spark_horovod/keras_spark_rossmann_estimator.py

```diff
@@ -92,11 +110,18 @@ class Hyperparameters:
     local_checkpoint_file: str = "checkpoint.h5"
     local_submission_csv: str = "submission.csv"
 
-
+# %%
+# We define a task that downloads the data into a Flyte directory.
 @task(
     cache=True,
     cache_version="0.1",
 )
+# %%
+# Downloading the data
+# ====================
+#
+# We use the `https://cdn.discordapp.com/attachments/545481172399030272/886952942903627786/rossmann.tgz`__ (Rossmann store sales data) to train our Keras model.
+# This function returns the path to the directory where the data is downloaded.
```

Comment:
I think we can remove this. We shouldn't be having it here (between decorator and function). And I think just saying that this task helps download the data should suffice.